src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -893,6 +893,63 @@ class CommonVisitor : public AST::BaseVisitor<Struct> {
         return true;
     }
 
+    int64_t find_argument_position_from_name(ASR::StructType_t* orig_struct, std::string arg_name) {
+        int64_t arg_position = -1;
+        for( size_t i = 0; i < orig_struct->n_members; i++ ) {
+            std::string original_arg_name = std::string(orig_struct->m_members[i]);
+            if( original_arg_name == arg_name ) {
+                return i;
+            }
+        }
+        return arg_position;
```

Comment:
I just noticed this. It seems `arg_position` is not needed here, we can just return `-1` at the end of the function. Sorry about it, I will update it in an upcoming PR.

Reply:
fixed here https://github.com/lcompilers/lpython/pull/1816.